pkg/apis/discovery/types.go

```diff
@@ -107,6 +107,10 @@ type Endpoint struct {
 	// with the EndpointSliceNodeName feature gate.
 	// +optional
 	NodeName *string
+	// hints contains information associated with how an endpoint should be
+	// consumed.
+	// +optional
```

Comment:
Add a `+featureGate=<gate>`  tag

Reply:
Updated, thanks!

Reply:
What is the effect of the `+featureGate=<gate>` tag?